checkers.kv

```diff
@@ -0,0 +1,64 @@
+<TitleMenu>:
+    BoxLayout:
+        orientation: 'vertical'
```

Comment:
As we're building for desktop shouldn't these be horizontal?

Reply:
In its current state horizontal menu buttons are a little awkward.  I'm sure all this stuff will get moved around as the app takes shape.